src/Common/src/Interop/OSX/Interop.Libraries.cs

```diff
@@ -4,7 +4,7 @@
 
 internal static partial class Interop
 {
-    private static partial class Libraries
+    internal static partial class Libraries
```

Comment:
That's because we want to compile src/Common/src/Interop/OSX/Interop.Libraries.cs, src/Common/src/Interop/Unix/Interop.Libraries.cs and src/Common/src/Interop/Windows/Interop.Libraries.cs, so it's to make the compiler happy.